build.wake

```diff
@@ -13,100 +13,103 @@
+    buildLSP,
+    buildJobCache,
+    buildWakeFormat,
+    Nil
+
 
 def all variant =
-    require Pass x =
-        map (_ variant) targets
-        | findFail
+    require Pass x = map (_ variant) targets | findFail
     Pass (flatten x)
 
+
 # Install wake into a target location
 def doInstall dest kind =
     require Pass variant = toVariant kind
-    require Pass datfiles =
-        sources "{here}/share" `.*`
+    require Pass datfiles = sources "{@here}/share" `.*`
     def binfilesResult = all variant
     def releaseBin exe = installAs "{dest}/{replace `\.[^.]*$` '' exe.getPathName}" exe
-    require Pass datinstall =
-        findFailFn (installIn dest ".") datfiles
-    require Pass readmeSource =
-        source "README.md"
-    require Pass readme =
-        installIn "{dest}/share/doc/wake" "." readmeSource
+    require Pass datinstall = findFailFn (installIn dest ".") datfiles
+    require Pass readmeSource = source "README.md"
+    require Pass readme = installIn "{dest}/share/doc/wake" "." readmeSource
     require Pass binfiles = binfilesResult
-    require Pass bininstall =
-        findFailFn releaseBin binfiles
+    require Pass bininstall = findFailFn releaseBin binfiles
     def outputs = readme, bininstall ++ datinstall
     Pass outputs
 
+
 # Replace @VERSION@ with 'release'
 def setVersion release file =
-    require Pass in =
-        source "{file}.in"
-    def script = "%
+    require Pass in = source "{file}.in"
+    def script =
+        "%
         set -e
         sed "s/@VERSION@/%{release}/g" "%{in.getPathName}" > "%{file}.tmp"
         mv "%{file}.tmp" "%{file}"
         %"
     shellJob script (in, Nil) | getJobOutput
 
+
 # Create a release tarball
 def tarball Unit =
     def releaseResult = buildAs Unit
     def timeResult = buildOn Unit
     require Pass release = releaseResult
     require Pass time = timeResult
     # Create debian + RedHat package files
-    require Pass changelog =
-        setVersion release "debian/changelog"
-    require Pass spec =
-        setVersion release "wake.spec"
+    require Pass changelog = setVersion release "debian/changelog"
+    require Pass spec = setVersion release "wake.spec"
     # Identify those sources files to include in the tarball
-    require Pass allSources =
-        sources "." `.*`
+    require Pass allSources = sources "." `.*`
     def srcs =
         allSources
         | filter (! matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
         | filter (! matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
         | filter (! matches `(.*/)*\.gitignore` _.getPathName) # omit git files
     def Pair testPaths wakePaths = splitBy (matches `tests/.*` _.getPathName) srcs
-    def wakeSourcesString =
-        wakePaths
-        | map (_.getPathName.format)
-        | foldr (_, ",\n  ", _) Nil
-        | cat
+    def wakeSourcesString = wakePaths | map (_.getPathName.format) | foldr (_, ",\n  ", _) Nil | cat
```

Comment:
We might want to consider a special exemption for `|` where we prefer to do it multiline even if it isn't super long. Perhaps just decrease the threshold in this case. (So really simple ones are still one line, but the example above still ends up multiline)

Reply:
Yeah, seems like that's pretty universally agreed upon. Def something we can do